src/test/java/org/sagebionetworks/common/util/progress/ThrottlingProgressCallbackTest.java

```diff
@@ -13,7 +13,7 @@
 public class ThrottlingProgressCallbackTest {
 	

```

Comment:
Need to remove the Throttle now that the heartbeat is the only call of progress made.